### ISSUE TITLE
Authenticode check failed unable to load Scintilla Library

### ISSUE THREAD

**Issue by dayton967** [notepad-plus-plus/notepad-plus-plus#3805]
### Description of the Issue
Upon Starting Notepad++

### Steps to Reproduce the Issue

1. Downloaded the ZIP for notepad++ 7.5.1
2. Ran Notepad++

### Expected Behavior
Should start notepad++

### Actual Behavior
Pop up for "Library Verification Failure" with the message "Authenticode Check Failure: signature or signing certificate are not recognized"
Followed by a Pop up  with "Exception On WM_CREATE" with the message "ScintillaEditView::init : SCINTILLA ERROR - Can not load the dynamic library"

### Debug Information
Notepad++ doesn't run.

![popup1](https://user-images.githubusercontent.com/4029174/31235186-cb0e2ffa-a9be-11e7-8fc3-493c4c7f9200.jpg)
![popup2](https://user-images.githubusercontent.com/4029174/31235187-ccaf41f0-a9be-11e7-9fe6-4033a2e6388f.jpg)

**Comment by LuciferSam86** [comment]
seems this reason: https://notepad-plus-plus.org/news/notepad-7.3.3-fix-cia-hacking-issue.html

are time and date correct on the system?
Did you check the hashes for the program here: https://github.com/notepad-plus-plus/notepad-plus-plus/releases/tag/v7.5.1
Did you try do download again NPP?

**Comment by dayton967** [comment]
The time is accurate. and I have tried to install a few times.
ff4b6e4fef1c1f8dd1f43337d26b9c711175ae58 *npp.7.5.1.bin.minimalist.7z

> signtool verify /v /pa "notepad++.exe"
Verifying: notepad++.exe

Signature Index: 0 (Primary Signature)
Hash of file (sha1): 3882FA198F4E83CF98156B94B6E8922BD377B0F2

Signing Certificate Chain:
    Issued to: DigiCert High Assurance EV Root CA
    Issued by: DigiCert High Assurance EV Root CA
    Expires:   Sun Nov 09 20:00:00 2031
    SHA1 hash: 5FB7EE0633E259DBAD0C4C9AE6D38F1A61C7DC25

        Issued to: DigiCert SHA2 High Assurance Code Signing CA
        Issued by: DigiCert High Assurance EV Root CA
        Expires:   Sun Oct 22 08:00:00 2028
        SHA1 hash: F7E0F449F1A2594F88856C0758F8E6F627E5F5A2


            Issued to: Notepad++
            Issued by: DigiCert SHA2 High Assurance Code Signing CA
            Expires:   Tue Jan 29 08:00:00 2019
            SHA1 hash: 9659849A76342C3DB71735F4C49449B29D453CA4

The signature is timestamped: Mon Aug 28 20:43:25 2017

Timestamp Verified by:
    Issued to: DigiCert Assured ID Root CA
    Issued by: DigiCert Assured ID Root CA
    Expires:   Sun Nov 09 20:00:00 2031
    SHA1 hash: 0563B8630D62D75ABBC8AB1E4BDFB5A899B24D43

        Issued to: DigiCert Assured ID CA-1
        Issued by: DigiCert Assured ID Root CA
        Expires:   Tue Nov 09 20:00:00 2021
        SHA1 hash: 19A09B5A36F4DD99727DF783C17A51231A56C117

            Issued to: DigiCert Timestamp Responder
            Issued by: DigiCert Assured ID CA-1
            Expires:   Mon Oct 21 20:00:00 2024
            SHA1 hash: 614D271D9102E30169822487FDE5DE00A352B01D

Successfully verified: notepad++.exe

Number of files successfully Verified: 1
Number of warnings: 0
Number of errors: 0


> signtool verify /v /pa "SciLexer.dll"
Verifying: SciLexer.dll

Signature Index: 0 (Primary Signature)
Hash of file (sha1): 1ADD5FA262E3BDDB57E7C996304D442C4AD849F9

Signing Certificate Chain:
    Issued to: DigiCert High Assurance EV Root CA
    Issued by: DigiCert High Assurance EV Root CA
    Expires:   Sun Nov 09 20:00:00 2031
    SHA1 hash: 5FB7EE0633E259DBAD0C4C9AE6D38F1A61C7DC25
	
        Issued to: DigiCert SHA2 High Assurance Code Signing CA
        Issued by: DigiCert High Assurance EV Root CA
        Expires:   Sun Oct 22 08:00:00 2028
        SHA1 hash: F7E0F449F1A2594F88856C0758F8E6F627E5F5A2

            Issued to: Notepad++
            Issued by: DigiCert SHA2 High Assurance Code Signing CA
            Expires:   Tue Jan 29 08:00:00 2019
            SHA1 hash: 9659849A76342C3DB71735F4C49449B29D453CA4

The signature is timestamped: Mon Aug 28 20:43:26 2017

Timestamp Verified by:
    Issued to: DigiCert Assured ID Root CA
    Issued by: DigiCert Assured ID Root CA
    Expires:   Sun Nov 09 20:00:00 2031
    SHA1 hash: 0563B8630D62D75ABBC8AB1E4BDFB5A899B24D43

        Issued to: DigiCert Assured ID CA-1
        Issued by: DigiCert Assured ID Root CA
        Expires:   Tue Nov 09 20:00:00 2021
        SHA1 hash: 19A09B5A36F4DD99727DF783C17A51231A56C117

            Issued to: DigiCert Timestamp Responder
            Issued by: DigiCert Assured ID CA-1
            Expires:   Mon Oct 21 20:00:00 2024
            SHA1 hash: 614D271D9102E30169822487FDE5DE00A352B01D


Successfully verified: SciLexer.dll

Number of files successfully Verified: 1
Number of warnings: 0
Number of errors: 0

**Comment by Sidewinder53** [comment]
I'm having the exact same issue.

**Comment by evpobr** [comment]
Why not build Scilexer as static library? Wikileaks issue fixed, relase builds are simplified, no certificate fails.

**Comment by jaycuse** [comment]
Having same issue with latest release 7.5.4

**Comment by JavaTec** [comment]
Having similar issue with 7.5.4 and 7.5.2. But 7.3.1 was installed without any issue

**Comment by c3h899** [comment]
Appears to be an issue for non-platform specific releases in atypical locations. Attempting to dump the x86 version in my Windows directory threw a similar issue, however installing the latest x64 build to the same location seems to have no issues. May have been simply fixed in latest patch, but doubtful.

**Comment by Lujaina95** [comment]
if you tried to debug the project and it shows to you the following message:
" ScintillaEditView::init : SCINTILLA ERROR can not load the dynamic library "
this message mean that you do not have scilexer.dll after you build scilexer.dll with boost
so,what you have to do?
-you need to back to VS2017 and click on project properties to tell the linker where to look for libraries to link to.
To access this setting:
right-click on the project name in the Solution Explorer window,
then Properties ? Linker ? General ? Additional Library Directories.
Enter <boost_path>\stage\lib
you need to change x64 to X86 which is in Visual studio because Notepad++ v7.5.4 (32-bit) it just work with x86
Also, you need to download SDK which you can find it at the following link:
https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk


these steps will be helpful for who use:
Notepad++ v7.5.4 (32-bit)
operating system x64
Windows 10
VS2017

**Comment by donho** [comment]
I believe it's not an issue anymore.

**Comment by raunilillemets** [comment]
I have a similar issue with Notepad++ 7.8.5:

Whenever I open a .txt file, I get the error:
"Authenticode Check Failure: signature or signing certificate are not recognized"
However, the Notepad++ still opened.

Reinstalling Notepad++ did not help.

Then I tried to uninstall 4 different plugins that I did not use. Left only one plugin: "Compare".

This made things worse. Now, when I open a .txt file, I get the error:
"Authenticode Check Failure: signature or signing certificate are not recognized"
AND
"ScintillaEditView::init : SCINTILLA ERROR - Can not load the dynamic library"

And so the Notepad++ won't start.

I can still open Notepad++ from Taskbar, just can't open .txt files directly.
